.gitignore

```diff
@@ -1,3 +1,4 @@
 _build/
 *.install
 *.merlin
+test
```

Comment:
I think you forgot to remove this line ?

Reply:
It doesn't hurt